obspy/core/utcdatetime.py

```diff
@@ -943,7 +948,8 @@ def __sub__(self, value):
             # see datetime.timedelta.total_seconds
             value = (value.microseconds + (value.seconds + value.days *
                      86400) * 10**6) / 1e6
-        return UTCDateTime(ns=self._ns - int(round(value * 1e9)))
+        # return UTCDateTime(ns=self._ns - int(round((value * 1e9))))
+        return UTCDateTime(ns=self._ns - int(round((value * 1e9))))
```

Comment:
This is not any different than before, right?

Reply:
No I changed this then ended up changing it back to better state intent. I will clean this up.